### PR TITLE
V6/UI wallet layouts with slots

### PR DESCRIPTION
**Pull request by olgakup**
### Devop

* \[x] Updated CHANGELOG.md
* \[x] Add PR label

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,29 +2,36 @@
...
-- update mobile tools [#2669](https://github.com/MyEtherWallet/MyEtherWallet/pull/2669)
-- update mobile how it works [#2668](https://github.com/MyEtherWallet/MyEtherWallet/pull/2668)
-- Update mobile access wallet [#2663](https://github.com/MyEtherWallet/MyEtherWallet/pull/2663)
-- Update mobile create wallet [#2661](https://github.com/MyEtherWallet/MyEtherWallet/pull/2661)
-- Finalize routes [#2556](https://github.com/MyEtherWallet/MyEtherWallet/pull/2556)
-- Update/Add landing, create wallet, access wallet pages [#2493](https://github.com/MyEtherWallet/MyEtherWallet/pull/2493)
-- Apply MEW components to dashboard [#2378](https://github.com/MyEtherWallet/MyEtherWallet/pull/2378)
-- Add MY TOKENS VALUE component for Dashboard [#2015](https://github.com/MyEtherWallet/MyEtherWallet/pull/2015)
-- Update my eth balance component chart [#1987](https://github.com/MyEtherWallet/MyEtherWallet/pull/1987)
+<<<<<<< HEAD
```
Comment (gamalielhere) [inline CHANGELOG.md:5]: remove missed conflicts

File: src/core/components/TheLayoutWalletGrid.vue [hunk 2]
```diff
@@ -0,0 +1,79 @@
+<template>
+  <!--
+    =====================================================================================
```
Comment (gamalielhere) [inline src/core/components/TheLayoutWalletGrid.vue:3]: 👏🏽 👏🏽 👏🏽

**Review by gamalielhere — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (olgakup) [reply CHANGELOG.md]: fixed in https://github.com/MyEtherWallet/MyEtherWallet/pull/2863/commits/4bcc3b0ea324edcf408e3d08b3da699983905fb4

File: src/views/components-wallet/TheWalletFooter.vue [hunk 3]
```diff
@@ -1,42 +1,71 @@
...
+                return-object
+                single-line
+              ></v-select>
+            </div>
+          </div>
+        </v-col>
+      </v-row>
     </v-container>
+    <!-- <v-container
+      class="py-6 px-10 d-block d-md-flex align-center justify-space-between"
```
[the remark this answers is not in the document]
Comment (olgakup) [reply src/views/components-wallet/TheWalletFooter.vue]: fixed in https://github.com/MyEtherWallet/MyEtherWallet/pull/2863/commits/1dcf353fed0d0845f957a3d947844c979df1894c

File: src/views/layouts-wallet/TheDeployContractLayout.vue [hunk 4]
```diff
@@ -1,18 +1,24 @@
...
 
 <script>
-import Network from '@/modules/network/ModuleNetwork';
+import TheLayoutWalletGrid from '@/core/components/TheLayoutWalletGrid';
+import Swap from '@/core/components/AppSwap';
 import ModuleContractDeploy from '@/modules/contract/ModuleContractDeploy.vue';
 
 export default {
-  name: '',
+  name: 'TheDeplyContractLao',
```
[the remark this answers is not in the document]
Comment (olgakup) [reply src/views/layouts-wallet/TheDeployContractLayout.vue]: fixed!

**Pull request merged**